### DISCUSSION TITLE
访问http://localhost:4000/显示被拒绝

### DISCUSSION THREAD

**Discussion by BOSS2356** [hexojs/hexo#5412] [Question / Answer] [answered]
![微信截图_20240129010713](https://github.com/hexojs/hexo/assets/138679535/b6a97f45-b939-4b79-8229-b2445063a9b5)
查看了很多帖子，例如启动windows服务，然后我想看看端口，结果端口没有4000，请问各位大佬，这到底咋回事啊，请教了身边的几个人，都没遇到过，本人初一，实在是不知道怎么解决了，多谢大家

**Comment by stevenjoezhang** [comment, marked as answer]
创建 Hexo 站点后，执行`npx hexo s`，在看到
```
INFO  Hexo is running at http://localhost:4000/ . Press Ctrl+C to stop.
```
之后，直接访问`http://localhost:4000/` 应该就可以看到网站了
